test/edu/umass/cs/gnsclient/client/singletests/ActiveCodeTest.java

```diff
@@ -42,6 +44,15 @@ public ActiveCodeTest() {
       }
     }
   }
+  
+  /**
+   *
+   */
+  @Test
+  public void test_000_ActiveCodeCreateGuids() {
```

Comment:
Minor: I would name this method slightly differently as it really isn't doing what it says. Given that DISABLE_ACTIVE_CODE is temporary, this is OK.